src/components/List/index.tsx

```diff
@@ -45,15 +46,32 @@ const List = <T extends {}>(props: Props<T>) => {
     selected,
     getItemValue,
     variablesClassName,
-    listItemCategory
+    listItemCategory,
+    multiselect
   } = props;
 
+  const selectedItemsList =
+    !multiselect || (multiselect && !Array.isArray(selected)) ? [selected] : selected;
+
+  const updatedSelectedItems = item => {
```

Comment:
please rename to 'getSelectedItems' because it is not updating anything, just getting the selected items